app/graphql/mutations/base_mutation.rb

```diff
@@ -1,6 +1,83 @@
 # frozen_string_literal: true
 
 module Mutations
+  # Helpers for user related mutations
+  module UserHelper
+    def current_user
+      @current_user ||= context[:current_user]
+    end
+  end
+
+  # Helpers for inventory service related mutations
+  module InventoryServiceHelper
+    include UserHelper
+
+    def inventory_host(id)
+      ::HostInventoryAPI.new(
+        id,
+        nil, # unknown hostname
+        current_user.account,
+        ::Settings.host_inventory_url,
+        nil # infer identity from account
+      ).inventory_host
+    end
+  end
+
+  # Helpers for host related mutations
+  module HostHelper
+    include UserHelper
+    include InventoryServiceHelper
+
+    def find_host(host_id, permission: :edit?)
```

Comment:
Looks like this method is not used anywhere anymore is it?

Reply:
Good catch - I've removed it.